### ISSUE TITLE
[2019] Pretraining Methods for Dialog Context Representation Learning

### ISSUE THREAD

**Issue by cfiken** [cfiken/paper-reading#28]
### タイトル
Pretraining Methods for Dialog Context Representation Learning
### リンク
https://arxiv.org/abs/1906.00414
### 著者
Shikib Mehri, Evgeniia Razumovskaia, Tiancheng Zhao, Maxine Eskenazi
### 投稿日付
ACL 2019
## 1. どんなもの？

対話文脈の表現学習ための事前学習(教師なし学習の目的関数)について調査した研究。
既存の事前学習モデルは言語モデルをベースにしており、token レベルでの表現を得られるが、対話では複数ターンの文脈が必要になる。既存の事前学習プロトコル2つに加えて、token レベルではなく discourse レベルの表現が学習できるように工夫を加えた提案手法2つの計4つの方法について実験、分析を行った。

## 2. 先行研究と比べてどこがすごい？

文レベルの理解が必要になる対話ドメインをターゲットとして、既存の事前学習プロトコルと、discourse レベルに特化するよう工夫を加えた事前学習プロトコルの系4つについて調査・実験・分析を行った。

## 3. 技術や手法のキモはどこ？

事前学習の目的関数として、次の4つを試す。
- Next-Utterance Retrieval (NUR)
  - 次の発話を候補の中から選択する
  - loss は候補の中での分類での cross entropy
- Next-Utterance Generation (NUG)
  - 次の発話を生成する
  - loss は各 token の確率の負の対数尤度 (= cross entropy)
- Masked-Utterance Retrieval (MUR)
  - MLM の文 ver. で、対話履歴の一つの文を replace して、正解を候補から当てる
  - loss は候補の中での分類での cross entropy
- Inconsistency Identification (InI)
  - MUR では一つの文を replace し、正解を候補の中から探すが、InI は replace された文がどれかをインデックスで当てる
  - loss はインデックスの分類での cross entropy

**Comment by cfiken** [comment]
- BERT とか既存との比較してほしい
- global representation と local representation をちゃんと意識してなかったので勉強になった
